### ISSUE TITLE
add `jmx` jq function to retrieve a MBean value

### ISSUE THREAD

**Issue by eiiches** [eiiches/java-prometheus-metrics-agent#3]
E.g. `jmx("java.lang:type=OperatingSystem"; "Version"}`.

I'd like to use this value in metric labels.

Related: #1

**Comment by eiiches** [comment]
`java.lang:type=Runtime:VmVersion` might also be useful.

**Comment by eiiches** [comment]
```yaml
labels: |
  {
    os_version: jmx("java.lang:type=OperatingSystem"; "Version"),
    jvm_version: jmx("java.lang:type=Runtime"; "VmVersion")
  }
```

**Comment by eiiches** [comment]
Fixed in 1b96eb1a21bbec84d68769051dcefd5e73df5024.
